mne/viz/epochs.py

```diff
@@ -371,6 +371,47 @@ def plot_epochs_trellis(epochs, epoch_idx=None, picks=None, scalings=None,
     fig : Instance of matplotlib.figure.Figure
         The figure.
     """
+    _plot_epochs_trellis(epochs, epoch_idx, picks, scalings, title_str,
+                         show, block, n_epochs)
```

Comment:
this fixes a problem where `plot_epochs(..., trellis=True)` was throwing a deprecationwarning when only `plot_epochs_trellis(...)` should have been, cc @jaeilepp